### DISCUSSION TITLE
Create and manage multiple desktop windows

### DISCUSSION THREAD

**Discussion by neel4os** [zauberzeug/nicegui#2528] [Ideas / Feature Requests]
### Question

I am trying to write a desktop application using nicegui. I would like to know if it is possible to create multiple windows (parent -child relationship with the main window) and configure each one independently. For example from wails documentation, they have provided the same capability (https://v3alpha.wails.io/whats-new/#multiple-windows)

**Comment by rodja** [comment]
This is currently not possible. But let's convert your Q&A to a feature request. Maybe someone would like to create a pull request?

**Reply by KiddoV** [reply]
May I ask why this is not possible since: NiceGUI v1.2 introduced a [native mode](https://nicegui.io/documentation#native_mode), which is powered by Pywebview. and Pywebview has [multiple windows](https://pywebview.flowrl.com/3.7/examples/multiple_windows.html) supports.

**Reply by rodja** [reply]
Pywebview unfortunately wants to be the "main process". Same as FastAPI. So we launch pywebview in its own process and have build a (quite complicated) bridge to let NiceGUI's event loop transparently interact with this other process. This bridge does not expose handing of multiple windows (yet). Be we welcome any pull requests in this regard.

**Comment by KiddoV** [comment]
Any news on this? 😁

**Reply by ecxchris** [reply]
https://github.com/zauberzeug/nicegui/discussions/5462
